src/main/java/bookface/logic/commands/ClearCommand.java

```diff
@@ -10,9 +10,12 @@
  */
 public class ClearCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "User list has been cleared!";
+    public static final String COMMAND_WORD = "clear all";
+    public static final String MESSAGE_SUCCESS = "All data has been cleared!";
 
+    public static final String MESSAGE_USAGE = Command.generateMessage(COMMAND_WORD, "Clears all "
```

Comment:
It looks more neat, but it may be arguable that its inconsistent for SLAP in comparison to other string messages.